### PR TITLE
Enable BitCoin SV GitPod.io deployment and build with one click

### PR DESCRIPTION
**Pull request by auxon**
Please give this a spin with GitPod.IO, and it will build a full BitCoin SV node in no time, no pain.

### CODE REVIEW

**Comment by Safer01** [comment]
Hi

**Comment by auxon** [comment]
No?

**Comment by auxon** [comment]
Hi

**Comment by auxon** [comment]
> Hi

Hi

File: .gitpod.dockerfile [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+RUN sudo add-apt-repository universe
+RUN sudo apt-get update
+RUN sudo apt-get install -y libboost-all-dev
+
+RUN sudo apt-get install -y libdb-dev
+RUN sudo apt-get install -y libdb++-dev
+
+# Build BitCoin
+RUN cd /workspace/bitcoin-sv/
+RUN ./autogen.sh
```
Comment (auxon) [inline .gitpod.dockerfile:16]: Kk

**Review by auxon — COMMENTED** [review]
Good

**Comment by pfromberg** [comment]
We do not support GitPod.io.

**Pull request closed without merging**